libs/novui/src/components/toggle/Toggle.tsx

```diff
@@ -0,0 +1,34 @@
+import { PolymorphicRef } from '../../types/props-helpers';
+import { ChangeEvent, forwardRef, ReactNode } from 'react';
+import { Switch } from '@mantine/core';
+import { css, cx } from '../../../styled-system/css';
+import { toggle } from '../../../styled-system/recipes';
+import { splitCssProps } from '../../../styled-system/jsx';
+
+interface IToggleProps {
```

Comment:
```suggestion
export type ToggleProps = CoreProps & {
```
Should be exported and we are now using type instead of interface. CoreProps has the `className`